mopidy/internal/network.py

```diff
@@ -425,6 +425,10 @@ def on_receive(self, message):
         if not self.prevent_timeout:
             self.connection.enable_timeout()
 
+    def on_failure(self, exception_type, exception_value, traceback):
```

Comment:
Should we log something in here at debug and/or warning level?

Reply:
The actual exception will have already been logged by the actor so I thought we didn't need anything more, just to clean up.

Reply:
I can provide a snippet showing what is logged when this triggers if that's helpful.

Reply:
Nah, that's fine, just wanted to make sure it was already logged.